api/src/metadata_json.rs

```diff
@@ -63,7 +94,7 @@ impl MetadataJson {
             .ok_or_else(|| anyhow!("no uri. call #upload before #save"))?;
 
         let metadata_json_active_model = metadata_jsons::ActiveModel {
-            collection_id: Set(collection),
+            id: Set(id),
```

Comment:
Do we have migration to move collection_id to id? Haven't gotten to the migrations yet.